diff --git a/platformtheme/CMakeLists.txt b/platformtheme/CMakeLists.txt
index cc55f01..5853b30 100644
--- a/platformtheme/CMakeLists.txt
+++ b/platformtheme/CMakeLists.txt
@@ -79,11 +79,7 @@ if(NOT QT_QMAKE_EXECUTABLE)
     message(FATAL_ERROR "qmake is not found.")
 endif()
 
-# execute the command "qmake -query QT_INSTALL_PLUGINS" to get the path of plugins dir.
-execute_process(COMMAND ${QT_QMAKE_EXECUTABLE} -query QT_INSTALL_PLUGINS
-    OUTPUT_VARIABLE QT_PLUGINS_DIR
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-)
+set(QT_PLUGINS_DIR ${KDE_INSTALL_QTPLUGINDIR})
 if(QT_PLUGINS_DIR)
     message(STATUS "Qt5 plugin directory:" "${QT_PLUGINS_DIR}")
 else()
diff --git a/widgetstyle/CMakeLists.txt b/widgetstyle/CMakeLists.txt
index cc55f01..5853b30 100644
--- a/widgetstyle/CMakeLists.txt
+++ b/widgetstyle/CMakeLists.txt
@@ -15,6 +15,7 @@ find_package(PkgConfig REQUIRED)
 find_package(KF5 REQUIRED WindowSystem)

+find_package(ECM 5.93.0)
 include(ECMQueryQmake)

 set (SRCS
     blurhelper.cpp
